docs/Getting-Started-TPU-VM.md

```diff
@@ -43,25 +45,24 @@ Defaults are:
 - `type`: `v3-32`
 - `preemptible`: `false`
 
-The command will spam you with a lot of output, sorry.
-
-In addition to creating the instance, it will set up the venv on each worker, and it will clone the repo to `~/levanter/`
-
-####
-If you use a preemptible instance, you probably want to use the "babysitting" script that automatically re-creates
-the VM. That's explained down below in the "Running Levanter GPT-2" section.
-
+**For Stanford CRFM Users**:
 
-### CRFM Setup
-
-Stanford CRFM folks can pass a different setup script to `infra/spin-up-tpu-vm.sh` to get our NFS automounted:
+Stanford CRFM folks can pass a different setup script to `infra/spin-up-vm.sh` to get our NFS automounted:
```

Comment:
should we maybe rename the script? WDYT?

Reply:
Do you mean renaming the shell script back to `spin-up-tpu-vm.sh`? I agree that's more specific. What was your intention when you renaming it to `spin-up-vm.sh`?